### PR TITLE
EX-SU-Pset_ElementAssemblyTypeRing & PSet_ArchElementTypeSegment

### PR DESCRIPTION
**Pull request by larswik**
Also added IfcElementAssemblyTypeEnum/RING

### CODE REVIEW

File: IFC4x3/Properties/l/LongitudinalJointConnections_0euxSzkhDAcfFPu$B$85Tb/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Specified if particular connector, eg. Guiding rod, is used.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LongitudinalJointConnections_0euxSzkhDAcfFPu$B$85Tb/Documentation.md]:
```suggestion
Specified if particular connector, e.g. guiding rod, is used.
```

File: IFC4x3/Properties/r/RadialJointConnections_0oSYw0FNv6_xS_Fj6q_a9u/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Specified if particular connector, eg. Dowell, is used.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RadialJointConnections_0oSYw0FNv6_xS_Fj6q_a9u/Documentation.md]:
```suggestion
Specified if particular connector, e.g. dowel, is used.
```

File: IFC4x3/Properties/s/SegmentType_3JQtIbIuv9iuE9bxPSMuVX/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="SegmentType" UniqueId="d36b74a5-4b8e-49b3-8389-97b65c5b87e1" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SegmentType_3JQtIbIuv9iuE9bxPSMuVX/DocProperty.xml]: No PrimaryDataType set.

File: IFC4x3/PropertyConstants/c/CONCAVE_CONVEX_2ce5d2r5bFARqt407gptBA.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONCAVE_CONVEX_2ce5d2r5bFARqt407gptBA" Name="CONCAVE_CONVEX" UniqueId="a6a059c2-d459-4f29-bd37-1001eacf72ca" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/c/CONCAVE_CONVEX_2ce5d2r5bFARqt407gptBA.xml]: Description.

File: IFC4x3/PropertyConstants/c/CONICAL_26RuieLNzFzgzGBCuts7cB.xml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONICAL_26RuieLNzFzgzGBCuts7cB" Name="CONICAL" UniqueId="866f8b28-557f-4ff6-af50-2cce37d8798b" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/c/CONICAL_26RuieLNzFzgzGBCuts7cB.xml]: Description.

File: IFC4x3/PropertyConstants/c/CONVEX_CONVEX_2aOB10Pv993OdO4W9RnUic.xml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONVEX_CONVEX_2aOB10Pv993OdO4W9RnUic" Name="CONVEX_CONVEX" UniqueId="a460b040-6792-490d-89d8-12025bc5eb26" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/c/CONVEX_CONVEX_2aOB10Pv993OdO4W9RnUic.xml]: Is this one properly named? Should it not be just CONVEX? Description.

File: IFC4x3/PropertyConstants/h/HEXAGONAL_2Jr9$rap1CU8tgt$57FRl1.xml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HEXAGONAL_2Jr9$rap1CU8tgt$57FRl1" Name="HEXAGONAL" UniqueId="93d49ff5-9330-4c78-8dea-dff1473dbbc1" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/h/HEXAGONAL_2Jr9$rap1CU8tgt$57FRl1.xml]: Description.

File: IFC4x3/PropertyConstants/i/INVERT_SEGMENT_WITH_SHOTCRETE_0ioCsK3H1CjgP5V2Oh9RYy.xml [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="INVERT_SEGMENT_WITH_SHOTCRETE_0ioCsK3H1CjgP5V2Oh9RYy" Name="INVERT_SEGMENT_WITH_SHOTCRETE" UniqueId="2cc8cd94-0d10-4cb6-a645-7c262b25b8bc" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/i/INVERT_SEGMENT_WITH_SHOTCRETE_0ioCsK3H1CjgP5V2Oh9RYy.xml]: Description.

File: IFC4x3/PropertyConstants/p/PARALLEL_3lJJ0s6nr5eOwnkdiq2IaG.xml [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PARALLEL_3lJJ0s6nr5eOwnkdiq2IaG" Name="PARALLEL" UniqueId="ef4d3036-1b1d-45a1-8eb1-ba7b34092910" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/p/PARALLEL_3lJJ0s6nr5eOwnkdiq2IaG.xml]: Description.

File: IFC4x3/PropertyConstants/p/PARALLEL_WITH_CORRECTIVE_RINGS_3CMdaJ0BDCQg7JquajrJyX.xml [hunk 10]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PARALLEL_WITH_CORRECTIVE_RINGS_3CMdaJ0BDCQg7JquajrJyX" Name="PARALLEL_WITH_CORRECTIVE_RINGS" UniqueId="cc5a7913-00b3-4c6a-a1d3-d3892dd53f21" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/p/PARALLEL_WITH_CORRECTIVE_RINGS_3CMdaJ0BDCQg7JquajrJyX.xml]: Description.

File: IFC4x3/PropertyConstants/s/SWISS_TÜBBING_3vXlScVwfBgv5kMIoxKGRM.xml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SWISS_TBBING_3vXlScVwfBgv5kMIoxKGRM" Name="SWISS_TÜBBING" UniqueId="f986f726-7faa-4bab-916e-592cbb5106d6" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/s/SWISS_TÜBBING_3vXlScVwfBgv5kMIoxKGRM.xml]: I would remove this as too specific. Description.

File: IFC4x3/PropertyConstants/t/TONGUE_GROOVE_3$cx1SJoTDuQBpEoCXtIYt.xml [hunk 12]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TONGUE_GROOVE_3$cx1SJoTDuQBpEoCXtIYt" Name="TONGUE_GROOVE" UniqueId="ff9bb05c-4f27-4de1-a2f3-3b2321dd28b7" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/t/TONGUE_GROOVE_3$cx1SJoTDuQBpEoCXtIYt.xml]: Description.

File: IFC4x3/PropertyConstants/u/UNIVERSAL_3Hj28uW759TQTfl6vNKcMr.xml [hunk 13]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="UNIVERSAL_3Hj28uW759TQTfl6vNKcMr" Name="UNIVERSAL" UniqueId="d1b42238-8071-4975-a769-bc6e575265b5" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/u/UNIVERSAL_3Hj28uW759TQTfl6vNKcMr.xml]: Description.

File: IFC4x3/PropertyEnumerations/s/PEnum_SegmentJointType.xml [hunk 14]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_SegmentJointType" Name="PEnum_SegmentJointType" UniqueId="a31fee3d-97fe-4548-a339-830178e29478">
+	<Documentation>Enumeration of the different types of joints for arch segments used for rings and arches.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/PropertyEnumerations/s/PEnum_SegmentJointType.xml]:
```suggestion
	<Documentation>Enumeration different joint types for arch segments used for rings and arches.</Documentation>
```

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Some of the constant descriptions can probably be left blank. Not pretty, but ...

File: IFC4x3/Properties/s/SegmentType_3JQtIbIuv9iuE9bxPSMuVX/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/s/SegmentType_3JQtIbIuv9iuE9bxPSMuVX/DocProperty.xml]: This file should be deleted. We decided to use IfcArchElementType/SEGMENT instead.

**Comment by larswik** [comment]
> Some of the constant descriptions can probably be left blank. Not pretty, but ...

I would prefer leaving definitions to @JontyDonn et.al.

File: IFC4x3/Properties/s/SegmentType_3JQtIbIuv9iuE9bxPSMuVX/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/s/SegmentType_3JQtIbIuv9iuE9bxPSMuVX/DocProperty.xml]: > This file should be deleted. We decided to use IfcArchElementType/SEGMENT instead.

Deleted in https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/856ac09196cf19c2f696cc353c217a840768c34b

**Comment by SergejMuhic** [comment]
pinging @JontyDunn

**Comment by larswik** [comment]
Ping @Michel-Rives: Can you look at descriptions for constants?

File: IFC4x3/PropertyConstants/c/CONCAVE_CONVEX_2ce5d2r5bFARqt407gptBA.xml (hunk 4 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/c/CONCAVE_CONVEX_2ce5d2r5bFARqt407gptBA.xml]: Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

File: IFC4x3/PropertyConstants/c/CONICAL_26RuieLNzFzgzGBCuts7cB.xml (hunk 5 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/c/CONICAL_26RuieLNzFzgzGBCuts7cB.xml]: Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

File: IFC4x3/PropertyConstants/c/CONVEX_CONVEX_2aOB10Pv993OdO4W9RnUic.xml (hunk 6 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/c/CONVEX_CONVEX_2aOB10Pv993OdO4W9RnUic.xml]: This is from the domain.

File: IFC4x3/PropertyConstants/h/HEXAGONAL_2Jr9$rap1CU8tgt$57FRl1.xml (hunk 7 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/h/HEXAGONAL_2Jr9$rap1CU8tgt$57FRl1.xml]: Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

File: IFC4x3/PropertyConstants/i/INVERT_SEGMENT_WITH_SHOTCRETE_0ioCsK3H1CjgP5V2Oh9RYy.xml (hunk 8 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/i/INVERT_SEGMENT_WITH_SHOTCRETE_0ioCsK3H1CjgP5V2Oh9RYy.xml]: Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

File: IFC4x3/PropertyConstants/p/PARALLEL_3lJJ0s6nr5eOwnkdiq2IaG.xml (hunk 9 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/p/PARALLEL_3lJJ0s6nr5eOwnkdiq2IaG.xml]: Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

File: IFC4x3/PropertyConstants/p/PARALLEL_WITH_CORRECTIVE_RINGS_3CMdaJ0BDCQg7JquajrJyX.xml (hunk 10 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/p/PARALLEL_WITH_CORRECTIVE_RINGS_3CMdaJ0BDCQg7JquajrJyX.xml]: Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

File: IFC4x3/PropertyConstants/s/SWISS_TÜBBING_3vXlScVwfBgv5kMIoxKGRM.xml (hunk 11 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/s/SWISS_TÜBBING_3vXlScVwfBgv5kMIoxKGRM.xml]: Kept in and added description in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

File: IFC4x3/PropertyConstants/t/TONGUE_GROOVE_3$cx1SJoTDuQBpEoCXtIYt.xml (hunk 12 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/t/TONGUE_GROOVE_3$cx1SJoTDuQBpEoCXtIYt.xml]: Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

File: IFC4x3/PropertyConstants/u/UNIVERSAL_3Hj28uW759TQTfl6vNKcMr.xml (hunk 13 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/u/UNIVERSAL_3Hj28uW759TQTfl6vNKcMr.xml]: Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)

**Comment by larswik** [comment]
@SergejMuhic : Added descriptions from @Michel-Rives. Please check/merge.

**Comment by larswik** [comment]
Ping @SergejMuhic : Can hopefully be merged now.

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**